### PR TITLE
feat(tutoriales): new llaves de github section

### LINKED ISSUE #214: Add GitHub Keys tutorial
We are missing this article translated to spanish 

https://guide.eoscostarica.io/docs/tutorials/github-keys

### PR DESCRIPTION
**Pull request by Edgar097**
### GH Issue

Resolves #214  

### Steps to test
1. Go to Llaves de GitHub
1. Check Content

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/tutoriales/llaves-github.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_label: Llaves de GitHub 
+---
+
+## ¿Por qué necesitamos Llaves de GitHub?
+
+Definamos primero a GitHub como un sistema de control de versiones distribuidas en línea basado en un navegador para aquellos desarrolladores de software que utilizan el sistema de control de revisiones Git. Git utiliza la autenticación de clave pública SSH y no una autenticación por contraseña.  Cada usuario del sistema tiene que generar una clave. 
+
+El `SSH` es también conocido como Secure Shell o Secure Socket Shell. [Más sobre SSH en GitHub](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-ssh). Las claves SSH se utilizan para autenticar conexiones seguras que se generan a través de un algoritmo criptográfico de clave pública, siendo los más comunes RSA o DSA. Se trata de una fórmula unidireccional que garantiza que la clave pública puede derivarse de la clave privada, pero que la clave privada no puede derivarse de la clave pública.
+
+## ¿Cómo generar una nueva clave SSH?
```
Comment (xavier506) [inline docs/tutoriales/llaves-github.md:13]: ## ¿Cómo generar una nueva clave SSH en windows?

**Review by xavier506 — APPROVED** [review]
LGTM

File: docs/tutoriales/llaves-github.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- Le pedirá un archivo para guardar la clave, sólo tiene que pulsar enter. Esto acepta la ubicación del archivo por defecto. 
+
+- En el indicador (shell), escriba una frase de contraseña segura. Por razones de seguridad debe ser lo más compleja posible.
+
+¡Ahora tienes una nueva clave SSH!
+
+## Cómo añadir su clave SSH al ssh-agent
+
+- Asegúrate de que el ssh-agent se está ejecutando [más información](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/working-with-ssh-key-passphrases) , o inícialo manualmente:
```
Comment (leisterfrancisco) [inline docs/tutoriales/llaves-github.md:43]: 👾 Empty space 👾
> /working-with-ssh-key-passphrases) ,

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/tutoriales/llaves-github.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+---
+
+- Añade tu clave privada SSH al ssh-agent. Si creó su clave con un nombre diferente, o si está añadiendo una clave existente que tiene un nombre diferente, sustituya `id_ed25519` en el comando por el nombre de su archivo de clave privada.
+
+```shell
+$ ssh-add ~/.ssh/id_ed25519
+```
+
+- Ahora necesitas añadir tu clave pública a tu cuenta de **GitHub** , para ello [sigue estos pasos](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account).
```
Comment (steph222) [inline docs/tutoriales/llaves-github.md:63]: Empty space!

>**GitHub**,

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]
Good work

**Pull request merged**